.github/workflows/unit-test-rust.yaml

```
@@ -41,6 +41,9 @@ jobs:
     needs: [detect-ci-container]
     container:
       image: ghcr.io/vdaas/vald/vald-ci-container:${{ needs.detect-ci-container.outputs.TAG }}
+      credentials:
+         username: ${{ secrets.PACKAGE_USER }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 9

(indentation)